code/queuedjobs/SqsScheduleRunnerJob.php

```diff
@@ -25,8 +25,10 @@ class SqsScheduleRunnerJob implements SqsIntervalTask {
 
 
     public function processScheduledJobs() {
-        $this->queuedJobService->checkJobHealth();
-        $this->queuedJobService->checkdefaultJobs();
+        $queue = array(QueuedJob::QUEUED, SqsQueuedJobExtension::TYPE_SCHEDULED);
```

Comment:
Looking at some of the issues that have been causing blocked queues, this should also check for QueuedJob::LARGE

Reply:
@nyeholt Added QueudJob::LARGE.